File: Examples/VisualBasic/Articles/WorkingWithCalculationEngine/SettingFormulaCalculationModeWorkbook.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Cells

Namespace Articles.WorkingWithCalculationEngine
    Public Class SettingFormulaCalculationModeWorkbook
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
            ' Create a workbook
            Dim workbook As New Workbook()

            ' Set the Formula Calculation Mode to Manual
            workbook.Settings.CalcMode = CalcModeType.Manual

            ' Save the workbook
            workbook.Save(dataDir & "output_out_.xlsx", SaveFormat.Xlsx)
            ' ExEnd:1
        End Sub
    End Class
End Namespace

File: Examples/CSharp/Articles/WorkingWithCalculationEngine/SettingFormulaCalculationModeWorkbook.cs
using System.IO;

using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Articles.WorkingWithCalculationEngine
{
    public class SettingFormulaCalculationModeWorkbook
    {
        public static void Run()
        {
            // ExStart:1
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
            // Create a workbook
            Workbook workbook = new Workbook();

            // Set the Formula Calculation Mode to Manual
            workbook.Settings.CalcMode = CalcModeType.Manual;

            // Save the workbook
            workbook.Save(dataDir + "output_out_.xlsx", SaveFormat.Xlsx);
            // ExEnd:1
        }
    }
}